demo/DemoArticles/update.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/* eslint-disable require-jsdoc */
+
+const fs = require('fs')
+const request = require('request')
```

Comment:
This comment isn't intended to sway you either way but here's an example Jon Robson did using node-fetch: https://gist.github.com/jdlrobson/a358f9ed079fbc9ebf18